Backport-Guide.md

```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
     $ git fetch origin
     ```
 
-    And check the list of current stable branches:
+    Check the list of stable branches:
 
     ```bash
     $ cd ${GOPATH}/src/github.com/kata-containers/runtime
```

Comment:
I know you didn't add it, but this second `cd` is superfluous I think.

Reply:
Agreed - fixed.